Network/Packets/AddPacketTypeRequest.cs

```diff
@@ -27,34 +28,47 @@
 //  You should have received a copy of the GNU Lesser General Public License
 //  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 // ***********************************************************************
+
 #endregion Licence - LGPLv3
 
 using Network.Attributes;
+using System;
+using System.Reflection;
 
 namespace Network.Packets
 {
     /// <summary>
-    /// Sending unknown packets needs at first a addPacketTypeRequest to
-    /// inform the endPoint about the new incomming packet.
+    /// Instructs the paired <see cref="Connection"/> to add all the
+    /// <see cref="Type"/>s in the given <see cref="Assembly"/>.
     /// </summary>
-    [PacketType(8)]
+    [PacketType(6)]
```

Comment:
Is there any reason why u did rearrange the mapping?

Reply:
It didnt fit with the documentation (some ancient, non-existent packets were mentioned?) so i changed it. It shouldnt affect anything though, and is now more consistent so i think its fine...

Reply:
Ok. I am fine with it. Was just wondering.